docs/release-notes-v17.5.md

```diff
@@ -6,34 +6,16 @@ This release of Percona Distribution for PostgreSQL is based on Percona Server f
 
 ## Release Highlights
 
-This release fixes [CVE-2025-1094](https://www.postgresql.org/support/security/CVE-2025-1094/), which closed a vulnerability in the `libpq` PostgreSQL client library but introduced a regression related to string handling for non-null terminated strings. The error would be visible based on how a PostgreSQL client implemented this behavior. This regression affects versions 17.3, 16.7, 15.11, 14.16, and 13.19. For this reason, version 17.3 was skipped.
+This release is based on and extends the functionality of [Percona Distribution for PostgreSQL 17.4.1](https://docs.percona.com/postgresql/17/release-notes-v17.4.html).
```

Comment:
17.5? We have PPG 17.4 based on PG 17.4. 
This is a new version. Or am I missing something?